src/components/molecules/UiRating/UiRating.vue

```diff
@@ -39,8 +39,8 @@
           :value="`${item.index}`"
           :name="ratingName"
           class="ui-rating__option"
-          @mouseover="hoverHandler($event, item.index)"
-          @mouseleave="hoverHandler($event, item.index)"
+          :onmouseover="(event: Event) => hoverHandler(event, item.index)"
+          :onmouseleave="(event: Event) => hoverHandler(event, item.index)"
```

Comment:
Q: why can't use `@mouseove`r? IMO we should use `@mousover` instead `:onmouseover`

Reply:
yeah, I'll return to `@mouseover`, thanks 👍